libraries/AP_Mount/AP_Mount_Siyi.cpp

```diff
@@ -956,4 +969,54 @@ bool AP_Mount_Siyi::get_rangefinder_distance(float& distance_m) const
     return true;
 }
 
+bool AP_Mount_Siyi::_check_firmware_version() const {
```

Comment:
Small nitpick but we normally don't prefix method names with an underscore.  Also let's put a comment above the method... I know it's pretty obvious what the method does but we always have comments so let's not stop now.  Something simple like, "check gimbal firmware version.  relies on the hardware and firmware version having been received from the gimbal.  returns true on success"

Also the curly bracket should be on the next line.

Reply:
Done.